docs/developer/binarywheels.rst

```diff
@@ -0,0 +1,94 @@
+Building binary wheels
+======================
+
+Several packages in the Zope Foundation repositories contain C code, which
+means they should be packaged and published as binary wheels alongside the
+standard egg archive. Building binary wheels has its pitfalls, though. If not
+done right they can cause application instability and crashes.
+
+
+.. warning::
+    Publishing binary wheels built in a local build environment is highly
+    discouraged! Every build environment is different, binary wheels made for
+    publication on PyPI should only be built in standardized and predictable
+    environments.
+
+To minimize problems we have automated binary wheel building using CI providers
+like Appveyor and GitHub Actions. Wheels are built and uploaded using a special
+GitHub account named ``zope.wheelbuilder``. These configurations and the PyPI
+account are currently maintained by Marius Gedminas, Michael Howitz and Jens
+Vagelpohl. You can contact them by posting your question or bug report in the
+`meta repository issue tracker
+<https://github.com/zopefoundation/meta/issues>`_. Here is a short description
+of the process, please note that some steps can only be performed by the
+configuration maintainers:
+
+Prerequisites
+-------------
+- every project publishing binary wheels must add the PyPI account
```

Comment:
Minor nit: All these bullet points seem to be sentences or paragraphs, but they don't start with a capital letter.
```suggestion
- Every project publishing binary wheels must add the PyPI account
```
etc